### ISSUE TITLE
how do you use this?

### ISSUE THREAD

**Issue by loordgek** [HenryLoenwind/waterhook#1]
"For details how to use Waterhook to prevent infinite water, see the API documentation in the source code."
where is the doc??

**Comment by HenryLoenwind** [comment]
Yeah, that sentence is a bit off. However, this is all anyone should need:

https://github.com/HenryLoenwind/waterhook/blob/master/src/main/java/info/loenwind/waterhooks/WaterHooksMod.java#L26-L30

;)
